app/middlewares/user.js

```diff
@@ -0,0 +1,36 @@
+const { checkSchema, validationResult } = require('express-validator');
+
+exports.dataUser = (req, res, next) => {
+  const errors = validationResult(req);
+  if (!errors.isEmpty()) {
+    return res.status(422).json({ errors: errors.mapped() });
+  }
+  return next();
+};
+
+exports.schemaUser = checkSchema({
+  password: {
+    exists: true,
+    errorMessage: 'Password is required',
+    isLength: {
+      errorMessage: 'Password should be at least 8 chars long',
+      options: { min: 8 }
+    }
+  },
```

Comment:
Here you can also add the `isAlphanumeric` property, also the idea of this scheme is that you do not have to do more validations in the controller, so those validations that are there no longer go.